airflow/models/taskinstance.py

```diff
@@ -594,6 +594,8 @@ def insert_mapping(run_id: str, task: "Operator", map_index: int) -> dict:
     @reconstructor
     def init_on_load(self):
         """Initialize the attributes that aren't stored in the DB"""
+        # correctly config the ti log
+        self._log = logging.getLogger("airflow.task")
```

Comment:
Since `init_on_load` is also called in `__init__`, we can remove the `getLogger` line in there.

Reply:
nice. updated. pls take a look.